Buildings/Controls/OBC/FDE/DOAS/EnergyWheel.mo

```diff
@@ -0,0 +1,276 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block EnergyWheel "This block commands the energy recovery wheel and associated bypass dampers."
+  parameter Real recSet(
```

Comment:
Change name to dTThrEneRec. Temperature difference threshold between outdoor air and return air temperature above which energy recovery is enabled